crates/apollo-compiler/src/diagnostic.rs

```diff
@@ -0,0 +1,117 @@
+use crate::FileId;
+use crate::NodeLocation;
+use crate::SourceMap;
+use ariadne::ColorGenerator;
+pub use ariadne::ReportKind;
```

Comment:
If possible I’d prefer ariadne not to be a public dependency so semver-breaking changes don’t need to propagate

Reply:
Good point, I wanted to call this Severity rather than Kind anyways, and having our own enum that we can convert to ariadne when ready would allow for that.